### ISSUE TITLE
PEARL: jaws manager blocks are missing units

### ISSUE THREAD

**Issue by rerpha** [ISISComputingGroup/IBEX#8078]
As a user I would like to have my jaws manager blocks for sample width/height to show the units. 

### acceptance criteria
- [ ] jaws manager is modified to allow mm as units for its common.template file, but still default to "" as to not change any other instrument. 
- [ ] pearl jaws manager opi is modified to remove the superfluous "mm" labels that are hardcoded

**Comment by rerpha** [comment]
jaws manager change: https://github.com/ISISComputingGroup/EPICS-motorExtensions/pull/60
opi change: https://github.com/ISISComputingGroup/ibex_gui/pull/1632
https://github.com/ISISComputingGroup/IBEX/pull/8079

**Comment by rerpha** [comment]
child of #6052
